.github/workflows/velox_be.yml

```diff
@@ -103,6 +103,22 @@ jobs:
             --backend-type=velox --benchmark-type=h --fixed-width-as-double --disable-aqe --off-heap-size=20g -s=1.0 --cpus=16 --iterations=1 \
           && java -Xmx5G -XX:ErrorFile=/var/log/java/hs_err_pid%p.log -cp target/gluten-it-1.0-SNAPSHOT-jar-with-dependencies.jar io.glutenproject.integration.tpc.Tpc \
             --backend-type=velox --benchmark-type=ds --fixed-width-as-double --off-heap-size=8g -s=0.1 --cpus=16 --iterations=1'
+      - name: Create Celeborn Cluster
```

Comment:
Can Celeborn work on Ubuntu22.04? Now the Unbun20.04 CI time is too long. https://stackoverflow.com/questions/51613897/how-can-i-get-ubuntu-version-using-java

Reply:
@jinchengchenghh Celeborn works on Ubuntu22.04. The long time of Ubuntu20.04 CI is mainly due to the newly added TPCDS and TPCH tests of Spark3.2 and Spark3.3 using Celeborn. About 20 minutes longer than before.

Reply:
Maybe we can move these new tests to Velox backend / velox-backend-ubuntu2204-test

Reply:
@jinchengchenghh  I have added tests for both Ubuntu 20.04 and Ubuntu 22.04. If Ubuntu 22.04 is enough, I can remove Ubuntu 20.04.  @zhztheplayer @zhouyuan cc

Reply:
> @jinchengchenghh I have added tests for both Ubuntu 20.04 and Ubuntu 22.04. If Ubuntu 22.04 is enough, I can remove Ubuntu 20.04. @zhztheplayer @zhouyuan cc

I am changing to make celeborn CI use 22.04 only in https://github.com/oap-project/gluten/pull/1294. Please feel free to comment if having any issue.